src/Wallabag/ImportBundle/Import/WallabagV1Import.php

```diff
@@ -123,6 +126,10 @@ public function setFilepath($filepath)
     protected function parseEntries($entries)
     {
         $i = 1;
+        /*
+         * Untitled in all languages from v1. This should never have been translated
+         */
+        $untitled = array('Untitled', 'Sans titre', 'podle nadpisu', 'Sin título', 'با عنوان', 'per titolo', 'Sem título', 'Без названия', 'po naslovu', 'Без назви');
```

Comment:
Since the v2 import will also get that, can't you also include an empty string ?